projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -189,16 +184,6 @@ function JetpackLikesMessageListener( event ) {
 			break;
 		}
 
-		case 'clickPostLike':
-			// Add or remove the wrapper `liked` class based on the total likes.
-			if ( data.total > 0 ) {
-				document.querySelector( `#${ data.id }` ).classList.add( 'liked' );
-			} else {
-				document.querySelector( `#${ data.id }` ).classList.remove( 'liked' );
-			}
```

Comment:
Just to confirm, these classes are used in the wpcom version:

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Swf%2Syvxrf%2Qerfg%2Qabwdhrel.wf%3Se%3Q993p9pso%231395%2Q1400-og

but we're not using that anymore, right?

Reply:
I'm not sure if I decoded the OG link correctly, but if I did, this is an instance of what I described in the PR description:

> There are `liked`/`like` styles on the `wpl-button` element inside the iframe, and they determine the type of the "star" icon that is displayed on the button. But that's something different.

We're talking about selectors like `.wpl-button a.like, .wpl-button a.liked`, aren't we? These buttons are fine. This PR touches only the `div.jetpack-likes-widget-wrapper` elements.